stdlib/types.pyi

```diff
@@ -123,7 +123,28 @@ class CodeType:
             freevars: Tuple[str, ...] = ...,
             cellvars: Tuple[str, ...] = ...,
         ) -> None: ...
-    if sys.version_info >= (3, 8):
+    if sys.version_info >= (3, 8) and sys.version_info < (3, 10):
+        def replace(
+            self,
+            *,
+            co_argcount: int = ...,
+            co_posonlyargcount: int = ...,
+            co_kwonlyargcount: int = ...,
+            co_nlocals: int = ...,
+            co_stacksize: int = ...,
+            co_flags: int = ...,
+            co_firstlineno: int = ...,
+            co_code: bytes = ...,
+            co_consts: Tuple[Any, ...] = ...,
+            co_names: Tuple[str, ...] = ...,
+            co_varnames: Tuple[str, ...] = ...,
+            co_freevars: Tuple[str, ...] = ...,
+            co_cellvars: Tuple[str, ...] = ...,
+            co_filename: str = ...,
+            co_name: str = ...,
+            co_lnotab: bytes = ...,
+        ) -> CodeType: ...
+    elif sys.version_info >= (3, 10):
```

Comment:
The following structure would be simplest:

```
if sys.version_info >= (3, 10):
    ...
elif sys.version_info >= (3, 8):
    ...
else:
    ...
```

Reply:
*Slaps hand on head*